spec/helpers/admin/claims_helper_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:claim_school) { schools(:penistone_grammar_school) }
   let(:current_school) { create(:school, :student_loan_eligible) }
 
-  describe "eligibility_answers" do
+  describe "#eligibility_answers" do
```

Comment:
Would you mind splitting these renames into a separate commit?

Reply:
Done.